src/main/java/com/github/dockerjava/api/DockerClient.java

```diff
@@ -90,6 +91,8 @@
 
     CreateImageCmd createImageCmd(@Nonnull String repository, @Nonnull InputStream imageStream);
 
+    LoadImageCmd loadImageCmd(@Nonnull InputStream imageStream);
```

Comment:
When this feature appeared in docker API? Could you place javadoc with `since` (example could be found through search) ?